pkg/frontend/nvme_namespace.go

```diff
@@ -117,13 +124,23 @@ func (s *Server) DeleteNvmeNamespace(_ context.Context, in *pb.DeleteNvmeNamespa
 		log.Printf("error: %v", err)
 		return nil, err
 	}
+	re := regexp.MustCompile(`nvmeSubsystems/(.+)/nvmeNamespaces/(.+)`)
+
+	matches := re.FindStringSubmatch(in.GetName())
+	var subsys, ns string
+	if len(matches) >= 3 {
+		subsys = matches[1]
+		ns = matches[2]
+	} else {
+		return nil, status.Error(codes.InvalidArgument, "invalid input parameters")
+	}
 	// Validate that a resource name conforms to the restrictions outlined in AIP-122.
-	if err := resourcename.Validate(in.Name); err != nil {
+	if err := resourcename.Validate(ns); err != nil {
```

Comment:
why checking for `ns` is better then `name` ? 
I think `resourcename.Validate()` works well also on `nvmeSubsystems/(.+)/nvmeNamespaces/(.+)` not just IDs

Reply:
resloved